Makefile

```diff
@@ -0,0 +1,14 @@
+REPO?=gcr.io/must-override
+IMAGE?=aws-encryption-provider
```

Comment:
I hate myself for asking this but, should this be `aws_encryption-provider` to match the repo/sig?

Reply:
I'm hoping we can change `aws_encryption-provider` to `aws-encryption-provider`

Reply:
ok, works for me :)

Reply:
Yes, I think it's OK to change.  I'll do it once we merge this as otherwise I don't know what happens to in-flight PRs